R/Design-methods.R

```diff
@@ -417,6 +417,19 @@ setMethod("simulate",
           data = thisData
         )
 
+        # Get the MTD estimate from the samples.
+        sample_mtd <- dose(
+          mean(object@nextBest@target),
+          model = object@model,
+          samples = thisSamples
+        )
+
+        # Create list with median MTD and CV.
+        additional_stats <- list(
+          median_mtd = median(sample_mtd),
+          cv_mtd = mad(sample_mtd) / median(sample_mtd)
+        )
+
         ## return the results
         thisResult <-
           list(
```

Comment:
With the explanation above it is probably ok now? Or should we rename `median_mtd` to avoid any confusion with the selected dose from the dose grid at the end of the trial? E.g. `sample_median_mtd`?